homeassistant/components/seventeentrack/sensor.py

```diff
@@ -111,242 +108,193 @@ async def async_setup_entry(
 ) -> None:
     """Set up a 17Track sensor entry."""
 
-    client = hass.data[DOMAIN][config_entry.entry_id]
+    coordinator: SeventeenTrackCoordinator = hass.data[DOMAIN][config_entry.entry_id]
+
+    @callback
+    def _async_create_remove_entities():
+        old_packages = coordinator.data.current_packages - set(
+            coordinator.data.live_packages.values()
+        )
+        packages_to_add = (
+            set(coordinator.data.live_packages.values())
+            - coordinator.data.current_packages
+        )
```

Comment:
You can create a set outside of this function with all the tracking numbers for which you already made sensors, that would simplify the coordinator and this function

Reply:
cool. Done